tests/dragon/utils/channel.py

```diff
@@ -54,6 +54,16 @@ def __init__(self, key: pathlib.Path) -> None:
 
         self._file_path.touch()
 
+    def __getstate__(self):
+        state = self.__dict__.copy()
```

Comment:
this is danger-town and should go away. This is caused by inclusion of the channel in the InferenceRequest.  Swapping those channels with descriptors will avoid this ticking time-bomb

Reply:
the danger town is gone and can't hurt us anymore!